mitosheet/src/mito/components/endo/ColumnHeaderDropdown.tsx

```diff
@@ -39,188 +38,66 @@ export default function ColumnHeaderDropdown(props: {
-                onClick={() => {
-                    props.setUIState(prevUIState => {
-                        return {
-                            ...prevUIState,
-                            currOpenTaskpane: {type: TaskpaneType.CONTROL_PANEL},
-                            selectedColumnControlPanelTab: ControlPanelTab.FilterSort
-                        }
-                    })
-                }}
+                {...getPropsForDropdownItem(ActionEnum.Rename_Column)}
+                title='Rename'
             />
-            <DropdownItem 
-                title='Change Dtype'
-                onClick={() => {
-                    props.setUIState(prevUIState => {
-                        return {
-                            ...prevUIState,
-                            currOpenTaskpane: {type: TaskpaneType.CONTROL_PANEL},
-                            selectedColumnControlPanelTab: ControlPanelTab.FilterSort
-                        }
-                    })
-                }}
-            />
-            <DropdownItem 
-                title='Fill NaN Values'
-                onClick={() => {
-                    const columnIndexesSelected = getColumnIndexesInSelections(props.gridState.selections);
-                    const columnIDsToFillNaN = columnIndexesSelected.map(colIdx => props.sheetData?.data[colIdx]?.columnID || '').filter(columnID => columnID !== '')
 
-                    props.setUIState(prevUIState => {
-                        return {
-                            ...prevUIState,
-                            currOpenTaskpane: {type: TaskpaneType.FILL_NA, startingColumnIDs: columnIDsToFillNaN},
-                        }
-                    })
-                }}
+            <DropdownSectionSeperator isDropdownSectionSeperator={true}/>
+
+            <DropdownItem {...getPropsForDropdownItem(ActionEnum.Filter)}/>
+            <DropdownItem
+                {...getPropsForDropdownItem(ActionEnum.SortAscending)}
+                title='Sort A to Z'
+                icon={<SortAscendingIcon aColor='black'/>}
             />
-            <DropdownSectionSeperator isDropdownSectionSeperator/>
-            <DropdownItem 
-                title='Format'
-                onClick={() => {
-                    props.setUIState(prevUIState => {
-                        return {
-                            ...prevUIState,
-                            currOpenTaskpane: {type: TaskpaneType.CONTROL_PANEL},
-                            selectedColumnControlPanelTab: ControlPanelTab.FilterSort
-                        }
-                    })
-                }}
-                disabled={!isNumberDtype(props.columnDtype)}
-                tooltip={!isNumberDtype(props.columnDtype) ? "Only number columns can be formatted currently" : undefined}
+            <DropdownItem {...getPropsForDropdownItem(ActionEnum.SortDescending)}
+                title='Sort A to Z'
+                icon={<SortDescendingIcon aColor='black'/>}
```

Comment:
Move these into the actions if possible

Reply:
I want to reuse this icon and change the colors by using this prop, so I'm keeping these as icon overrides for now.